float64[12] x0
float64 t0
collocation_optimization_trajectory_planner/init_pos init_stance_pos
float64[2] vel_des
float64 yaw_rate_des
